projects/core/src/lib/operators/dialogs/expression-operator/expression-operator.component.html

```diff
@@ -1,7 +1,7 @@
 <geoengine-sidenav-header>Calculate Expression on Raster</geoengine-sidenav-header>
 
```

Comment:
"-" "+" icons in operator input not centered (on hover).

![image](https://user-images.githubusercontent.com/1027102/210562206-d649fb65-4c43-4688-92a5-f28a3001fb7a.png)

Reply:
![image](https://user-images.githubusercontent.com/4947061/211202322-2f2e6ab6-47b6-44e2-ba17-e636c070d817.png)

For me, it looks like if they are.

Reply:
![Peek 2023-01-09 10-20](https://user-images.githubusercontent.com/1027102/211275589-4f2baef2-2659-42ce-8a78-47a6fc66feee.gif)

Reply:
I now use icon buttons, they seem appropriate. I guess other buttons are designed to have at least some text next to the icon.

[6081ec4](https://github.com/geo-engine/geoengine-ui/pull/555/commits/6081ec4f8839694c35f576e999239086f1826c1c)